.github/workflows/mirror.yaml

```diff
@@ -54,7 +54,11 @@ jobs:
           curl https://mirrors.tuna.tsinghua.edu.cn/jenkins/updates/stable-2.107/update-center.json -o official/stable-2.107/update-center.json --create-dirs
           curl https://mirrors.tuna.tsinghua.edu.cn/jenkins/updates/stable-1.651/update-center.json -o official/stable-1.651/update-center.json --create-dirs
           curl https://mirrors.tuna.tsinghua.edu.cn/jenkins/updates/dynamic-stable-2.222.4/update-center.json -o official/dynamic-stable-2.222.4/update-center.json --create-dirs
+          curl https://mirrors.tuna.tsinghua.edu.cn/jenkins/updates/dynamic-stable-2.235.5/update-center.json -o official/dynamic-stable-2.235.5/update-center.json --create-dirs
+          curl https://mirrors.tuna.tsinghua.edu.cn/jenkins/updates/dynamic-stable-2.235.4/update-center.json -o official/dynamic-stable-2.235.4/update-center.json --create-dirs
```

Comment:
Hi @yJunS, I found out an error. `https://mirrors.tuna.tsinghua.edu.cn/jenkins/updates/dynamic-stable-2.235.4/update-center.json` does not exist.